src/pages/Home.vue

```diff
@@ -251,18 +250,53 @@ export default {
     <section
       :class="[
         'hero text-center',
-        style.homeHeroBase64 || isBackgroundLoaded ? 'text-white' : ''
+        (style.homeHeroBase64 || style.homeHeroUrl) && !style.homeHasLightBackground ? 'text-white' : ''
       ]"
-      :style="`background-image: url('${style.homeHeroBase64}')`"
+      :style="`background-image: url(${style.homeHeroBase64})`"
     >
-      <QImg
-        v-if="$q.screen.height > 720 && $q.screen.gt.xs"
-        class="hero__background absolute-full"
-        :src="homeHeroUrlTransformed"
-        no-default-spinner
-        basic
-        @load="isBackgroundLoaded = true"
-      />
+      <div class="hero__background absolute-full">
+        <picture>
+          <source
+            type="image/webp"
+            :srcset="`${
```

Comment:
Is it worth to create a util to handle the responsive srcset?
```js
function getImgResponsiveSrcset (widths, getUrl) {
  return widths
    .map(width => `${getUrlFn(width) ${width}w`})
    .joins(', ')
}

// Usage:
getImgResponsiveSrcset([1024, 1366, 1600, 1920, 2560], (width) => getHomeHeroUrlTransformed({ width }))
```

Reply:
Yeah, DRYer indeed, but I wanted to keep breakpoints in HTML for easier authoring (closer to final HTML output).

Repetition in 2 `<source>` tags will be removed when we update our AWS image handler to new version supporting `AUTO_WEBP`.